backend/src/services/secret-v2-bridge/secret-v2-bridge-service.ts

```diff
@@ -2159,15 +2173,19 @@ export const secretV2BridgeServiceFactory = ({
       actorOrgId,
       actionProjectType: ActionProjectType.SecretManager
     });
-    ForbiddenError.from(permission).throwUnlessCan(ProjectPermissionActions.Read, ProjectPermissionSub.SecretRollback);
+    ForbiddenError.from(permission).throwUnlessCan(ProjectPermissionCommitsActions.Read, ProjectPermissionSub.Commits);
```

Comment:
hmm can we do an OR condition here? just realized that some teams might already be using this `Secret Rollback` subject. This is an existing one so maybe let's just do that to prevent anything from breaking. Thoughts?